src/os/bluestore/BlueStore.cc

```diff
@@ -7964,16 +8072,29 @@ bluestore_deferred_op_t *BlueStore::_get_deferred_op(
 
 void BlueStore::_deferred_queue(TransContext *txc)
 {
-  dout(20) << __func__ << " txc " << txc << " on " << txc->osr << dendl;
+  dout(20) << __func__ << " txc " << txc << " osr " << txc->osr << dendl;
   std::lock_guard<std::mutex> l(deferred_lock);
```

Comment:
Just some thoughts I want to share - having such a common lock might probably impact the performance negatively - it seems the only global entity really needed such a protection is deferred_queue itself. Other stuff is per-OSR and hence might have another lock with narrower scope. May be refactor later?